openmapflow/labeled_dataset.py

```diff
@@ -79,53 +320,204 @@ class LabeledDataset:
     - The earth observation data for that coordinate (X)
     Together labels (y) and the associated earth observation data (X) can be used
     to train and evaluate a macine learning model a model.
-
-    Args:
-        dataset (str): The name of the dataset.
-        country (str): The country of the dataset (can be 'global' for global datasets).
     """
 
-    dataset: str = ""
-    country: str = ""
-    label_type: str = "binary"
-    license: str = ""
-    source: str = ""
-    dataset_dir: Path = PROJECT_ROOT / dp.DATASETS
-
     def __post_init__(self):
-        self.df_path = self.dataset_dir / (self.dataset + ".csv")
+        np.random.seed(SEED)
```

Comment:
Do we want to be reseeding stuff every time a class is initialized? 

Or do we want to initialize stuff once on initial import? with some global `HAS_BEEN_SEEDED` variable

Reply:
Moved to top of file